content/docs/manuals/extending-oakestra/create_addon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Create an Addon"
+summary: ""
+draft: false
+weight: 1
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
```

Comment:
There is a short description needed on "Addons" and what can you package as an addon. Highlight why this is a powerful functionality that makes Oakestra future-proof

Reply:
The suggestion feels more like a `concept` thing than a manual, no?